### ISSUE TITLE
Synchronize directories - the vertical scroll bar will not update the content of the list view when dragged with the mouse

### ISSUE THREAD

**Issue by VictorTrifu** [doublecmd/doublecmd#1855]
### Version

Double Commander
Version: 1.1.18 gamma
Revision: 568
Commit: c45365d1b
Build date: 2024/09/21
Lazarus: 3.5.0.0
FPC: 3.2.2
Platform: x86_64-Win64-win32/win64
OS version: Windows 10 Home 22H2 x86_64

### Description

After performing a compare operation in synchronized directories dialog, the resulted list has a right scroll bar . Dragging on this scroll bar with the mouse will not update the content of the list. 

Scrolling works fine when rotating the mouse wheel. You can also scroll by selecting one line and pressing page down multiple times. But this is very inconvenient when having very long lists.

Thank you for your effort to fix this bug. Double commander is very cool.

**Comment by VictorTrifu** [comment]
Again thank you so much for fixing that. I can't wait that the version  v1.1.19 is out.

**Comment by JonnyTech** [comment]
> I can't wait that the version v1.1.19 is out.

FYI if you want to have the fix sooner, you can try a [snapshot](https://doublecmd.sourceforge.io/site/eng/snapshots.php) which is automatically built after code commits:

```
Revision 11494 
changelog.txt
62dd19e08 alexx2000 2024-09-22 22:10:15 +0300
FIX: Synchronize directories - the vertical scroll bar will not update the content of the list view when dragged with the mouse (fixes #1855)
```

**Comment by VictorTrifu** [comment]
Thank you very much JonnyTech!
I have tested it and it works like a charm!
